typegen/strats/gen.py

```diff
@@ -0,0 +1,64 @@
+import abc, ast
+import pathlib
+import typing
+
+import pandas as pd
+
+import constants
+
+
+class Generator(abc.ABC):
```

Comment:
Choose a more precise name, for example, TypeHintGenerator, TypegenFileGenerator, ...

Reply:
`TypeHintGenerator` is good